src/components/GDPRConsent.js

```diff
@@ -10,33 +10,80 @@ import React, { useEffect } from 'react';
 import CookieConsent from 'react-cookie-consent';
 import TagManager from 'react-gtm-module';
 import { GTM_ID } from '../constants';
+import { useNewUiEnabled } from '../hooks';
 
 const GDPRConsent = () => {
+  const newUiEnabled = useNewUiEnabled();
+
+  const style = {
```

Comment:
suggestion(non-blocking): Minor suggestion to make the variable clearer
```suggestion
  const newUiStyle = {
```